src/main/java/uk/ac/ic/wlgitbridge/server/GitBridgeServer.java

```diff
@@ -50,7 +51,22 @@ public GitBridgeServer(Config config) throws ServletException, InvalidRootDirect
         this.rootGitDirectoryPath = config.getRootGitDirectory();
         RepoStore repoStore = new FSRepoStore(rootGitDirectoryPath);
         DBStore dbStore = new SqliteDBStore(repoStore.getRootDirectory());
-        SwapStore swapStore = new SwapStore() {};
+        SwapStore swapStore = new SwapStore() {
+            @Override
+            public void upload(String projectName, InputStream uploadStream, long contentLength) {
```

Comment:
This is something I'm not used to.  We're creating an anonymous empty implementation of the `SwapStore` interface here, right?  Is this something that's actually OK to do, or is this just a placeholder until it's replaced with a `SwapStore.fromConfig` call in a later commit?

(Pardon the question – I've been away from Java for some years, and I'm not sure this was possible in the Java I used back then.)